docs/changelog.md

```diff
@@ -11,6 +11,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/main/.config.yml)
 
+* **feature-addons**
+  * 🚒 Bug fixed [issue #11819] https://github.com/mozilla-mobile/android-components/issues/11819) - Remove nativeMessaging permission from the add-on BLOCKED_PERMISSIONS list.
```

Comment:
A more accurate/meaningful way of describing the changes is:

> Do not block installation of extensions with the "nativeMessaging" permission any longer, but ignore the permission instead for non-builtin add-ons.

Reply:
👍🏽

Reply:
Updated!